### DISCUSSION TITLE
Feature of the tooltip for the from address and the favicon

### DISCUSSION THREAD

**Discussion by lieser** [lieser/dkim_verifier#315] [Ideas]
As the from tooltip is currently broken in TB 102 (#311), I want to take this as an opportunity to have a general discussion about the feature of the tooltip for both the from address and the favicon.
The reason why I would like to remove them long term is to reduce the changes done to Thunderbird's own GUI (via experiments). This reduces the chance of something breaking with newer Thunderbird versions.

For that I would like to better understand why people use this features, and what possible alternatives would be.

The only real usage I see is if the DKIM header row is disabled to save space. The general status can then still be seen using the highlighting of the from header, but details are only visible in the tooltip.
An alternative for that could be to show the DKIM result then clicking the DKIM button in the header, there currently only the buttons for e.g. updating a saved key are visible. See also #160.

If you are actively using the tooltip please:
- Give feedback if the above alternative would work for you.
- If the above would not the a good alternative for you, describe your use case and why the alternative would not work.
- Possibly suggest a different alternative. Keep in mind that the goal is to reduce the changes done to Thunderbird's GUI.

**Comment by Albirew** [comment]
Another alternative can be to use the addon button tooltip, but both are fine as long as it doesn't eat a line in headers.

**Reply by lieser** [reply]
The addon button tooltip only allows a text to be set. So not really possible to show there a structured DKIM result.

**Comment by OhSoGood** [comment]
To avoid to eat a line in headers is important for me.
I'm using highlighting of the from header, and details in tooltip would be welcome (if it was working). To have the info in the statusbar would also be welcomed (as specified in https://github.com/lieser/dkim_verifier/issues/237 )

Thank you for the extension by the way!!!

**Reply by lieser** [reply]
Note that the tooltip is now fixed in master.

As for the statusbar I think it is unlikely that it will come back.
